keyboards/kbdfans/kbd8x_mk2/keymaps/via/rules.mk

```
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```

This compiles fairly close to the chip's capacity; let's enable Link Time Optimization.

```suggestion
VIA_ENABLE = yes
LTO_ENABLE = yes
```